src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -36,10 +41,24 @@ class ConfirmationPage extends React.Component {
     const { form, userSignedIn } = this.props;
     // If someone refreshes this page after submitting a form and it loads
     // without an empty response object, we don't want to throw errors
+    const {
+      first: firstName = '',
+      middle: middleName = '',
+      last: lastName = '',
+      suffix = ''
+    } = form.data.veteranFullName;
+
+    const { serviceBranch, verified } = form.data;
+
+    const photoUrl = getImageUrl(form.data.photo);
+
     const response = form.submission.response || {};
-    const { veteranFullName, verified } = form.data;
     const submittedAt = moment();
 
+    const veteranFullNameStr =
+      `${firstName.toUpperCase()} ${middleName.toUpperCase()} ${lastName.toUpperCase()} ${suffix.toUpperCase()}` // upper case name
```

Comment:
Opted for passing the whole name string in rather than the name parts even though the veteranID card breaks them back down into parts. The number of name parts vary (2 minimum up to 3 typically but can be higher with space separated double-barrel last names) so I thought it'd be simpler if the ID component could just assume a space splittable string since the which name part they are isn't relevant to it.